admin/src/scenes/support-center/ticket/create.js

```diff
@@ -73,6 +73,11 @@ export default () => {
                 touched={touched}
                 options={options}
               />
+              {(user.role === "referent_department" || user.role === "referent_region") && values.type === "Aide sur un cas particulier" && (
```

Comment:
il faudrait utiliser la constant `ROLES`
```suggestion
              {(user.role === ROLES.REFERENT_DEPARTMENT || user.role === ROLES.REFERENT_REGION) && values.type === "Aide sur un cas particulier" && (
```